plugins/formal-verification/formver.annotations/src/org/jetbrains/kotlin/formver/plugin/Builtins.kt

```diff
@@ -5,6 +5,9 @@
 
 package org.jetbrains.kotlin.formver.plugin
 
+private class FormverFunctionCalledInRuntimeError(offendingFunction: String) :
```

Comment:
I've been told that `Exception` should be preferred, as `Error` is only meant for very fatal things.